codenames-web-services/src/main/java/com/codenames/controllers/RoomsRestController.java

```diff
@@ -0,0 +1,47 @@
+package com.codenames.controllers;
+
+
+import com.codenames.models.game.CodeNamesGame;
+import com.codenames.models.game.Player;
+import com.codenames.models.game.User;
+import com.codenames.services.GameService;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/room")
+@CrossOrigin(origins = "http://26.193.49.236:3000")
```

Comment:
You shouldn't hardcode values specific to your system ex. IP address :)

Reply:
Reworked for configuration

Reply:
The IP is still here BTW :)

Reply:
Yes, you're right, the old version of the file has been saved